module/pom.xml

```diff
@@ -232,14 +261,14 @@
     <dependency>
       <groupId>org.mule.module</groupId>
       <artifactId>mule-java-module</artifactId>
-      <version>1.2.10</version>
+      <version>1.3.0-SNAPSHOT</version>
```

Comment:
Please use properties for these versions

Reply:
changed